packages/wrangler/.eslintrc.js

```diff
@@ -8,6 +8,7 @@ module.exports = {
 		"templates",
 		"emitted-types",
 		"e2e/seed-files/**/*",
+		"e2e/unenv-preset/**/*",
```

Comment:
Why are we ignoring these tests files?
Can we not just fix up any linting problems now?

Reply:
Because eslint would complain that the files are not covered by the tsconfig.
unenv needs a different config to be able to resolve imports (`module`, `moduleResolution`), which the PR adds a distinct `tsconfig.json`.